console/settings.py

```diff
@@ -23,6 +23,7 @@
 SECRET_KEY = os.getenv('CONSOLE_SECRET_KEY', 'secretwords')
 SECURITY_PASSWORD_SALT = os.getenv('CONSOLE_PASSWORD_SALT', '1ab')
 CONSOLE_LOGIN_TIMEOUT = os.getenv('CONSOLE_LOGIN_TIMEOUT', 10)
+CONSOLE_INITIAL_ADMIN_PASSW = os.getenv('CONSOLE_INITIAL_ADMIN_PASSW')
```

Comment:
That's a bit of an odd name for the var - no reason to truncate `PASSWORD` down to `PASSW`.

Reply:
Required environment envs should also be listed in the README file (see https://github.com/ONSdigital/sdx-store/blob/master/README.md)